runtests.py

```diff
@@ -826,8 +858,8 @@ def run_distutils(self, test_directory, module, workdir, incdir,
             ext_compile_flags = CFLAGS[:]
             compiler = COMPILER or sysconfig.get_config_var('CC')
 
-            if self.language == 'c' and compiler == 'gcc':
-                ext_compile_flags.extend(['-std=c89', '-pedantic'])
+            if self.language == 'c' and 'gcc' in compiler:
+                ext_compile_flags.append('-pedantic')
```

Comment:
In any case this seems to be what caused the build to fail--all the `long long` warnings cause the log length limit of 4MB to be exceeded and the job is terminated prematurely.